org_qubes_os_initial_setup/ks/qubes.py

```diff
@@ -104,26 +140,21 @@ def __init__(self, name):
         self.usbvm = self.usbvm_available
         self.usbvm_with_netvm = False
 
+        self.custom_pool = False
+        self.vg_tpool = None
+
         self.skip = False
 
+        self.default_template = None
+        self.templates_to_install = \
+            ['fedora', 'debian', 'whonix-gw', 'whonix-ws']
+
```

Comment:
All the settings here needs to be also properly serialized/deserialized in kickstart file. See `__str__` and `handle_line` methods.